src/util/heuristic.jl

```diff
@@ -4,7 +4,7 @@ function run_rop_heuristic(network_data, model_constructor, optimizer; heuristic
     return run_forward_restoration(network_data, model_constructor, optimizer; kwargs...)
 end
 
-
+# TODO update time_elapsed for based on the repairs
```

Comment:
If this is still TODO, we should add an issue to the issue tracker.

Reply:
Addressed now that the time_elapsed is calculated in the build network stage